portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,102 @@
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>
+        <intro-title> Hello, my name is Sydney!</intro-title>
+      </h1>
+      <br>
+      <greeting-button onclick="addRandomGreeting()">Inspiration</greeting-button>
       <div id="greeting-container"></div>
+      <h2>
+        <cursive>About Me</cursive>
+      </h2>
+        <p>I am a rising junior at Xavier University of Louisiana and majoring in Computer Science and Spanish.
+           Computer Science, because I love creating and designing new things. Spanish, because I love languages
+           and cultures (I can't wait to learn more about the world)! Once I graduate, I would like to work as a
+           User Experience Designer so that I can help people through my visual designs.
+        </p>
+        <p>I was born and raised in Augusta, Georgia, home of James Brown and the Masters Tournament. However,
+           the majority of my family is from Louisiana. My hobbies include traveling, drawing, painting, playing the guitar,
+           and watching TV. I also enjoy learning about psychology and personality traits. I'm an INFJ, and like
+           to classify myself as an introverted extrovert and a procrastinating perfectionist.
+        </p>
+        <hr>
+        <h2>
+          <cursive>Projects</cursive>
+        </h2>
+        <h3>
+          <cursive>Analytics World Map | UX and Website Design</cursive>
+        </h3>
+        <p>During my EP internship, I implemented styling and interaction of a map displaying the geographical
+           breakdown of end users. I also worked closely with UX to implement pixel-perfect frontend components
+           according to mocks.
+        </p>
+        <h3>
+          <cursive>Notebook | Applicatioin Development</cursive>
+        </h3>
+        <p>During my time at CSSI, I coded on a team of students to create a scrapbook application. Using Processing,
+           we implemented music, photos, and fun stickers.
+        </p>
+        <h3>
+          <cursive>Coffee Science | Application Design</cursive>
+        </h3>
+        <p>I designed an application for a coffee shop in New Orleans. I collected data on the customers and designed
+           the application based on customer feedback.
+        </p>
+        <h3>
+          <cursive>Art Meets Passion | Website Design</cursive>
+        </h3>
+        <p>For my senior project in high school, I partnered with InnovationV and designed a website for one of
+           their clients.
+        </p>
+        <hr>
+
+        <h2>
+            <cursive>Places I Would Love to Visit</cursive>
+        </h2>
+        <div class="gallery">
+            <img src="https://specials-images.forbesimg.com/imageserve/1182337590/960x0.jpg?fit=scale" alt="Bogota, Colombia" width="350" height="250">
```

Comment:
Fix indentation here and through the rest of the file.

Reply:
I don't think i'm familiar with the indentation, how should it look?

Reply:
Certainly! Basically, you've just used 4 spaces but should use 2. :) 2 spaces are used to show a parent/child relationship, and 4 spaces are used when you're continuing a long line to stay within the allowed line length.

Reply:
Gotcha!